modules/ROOT/pages/partner-manager-maps copy.adoc

```diff
@@ -0,0 +1,12 @@
+= Maps
```

Comment:
What ABOUT maps? What kind of maps? Can we provide a more specific title to the topic set?